app/ldap/server/ldap.js

```diff
@@ -484,7 +479,6 @@ export default class LDAP {
 	}
 
 	authSync(dn, password) {
```

Comment:
Did you escape the parameters that use this function where they are used?

Reply:
@MarcosSpessatto there is no user input from where this method is called. Only on login.